android/app/build.gradle

```diff
@@ -101,6 +117,19 @@ android {
             proguardFiles getDefaultProguardFile("proguard-android.txt"), "proguard-rules.pro"
         }
     }
+
+    flavorDimensions "buildtype"
+    productFlavors {
+        dev {
+            dimension "buildtype"
+            resValue "string", "app_name", project.env.get("APP_DISPLAY_NAME")
+            applicationIdSuffix ".dev"
+        }
+        prod {
+            dimension "buildtype"
+            resValue "string", "app_name", project.env.get("APP_DISPLAY_NAME")
```

Comment:
i guess that applicationIdSuffix is not necessary for prod ?

Reply:
Exactly, the idea is to have the application id as it goes since is the production env